src/AssociationRegistry/Vereniging/ContactGegevens/ContactgegevenType.cs

```diff
@@ -2,9 +2,11 @@
 
 public sealed class ContactgegevenType : IEquatable<ContactgegevenType>
 {
+    public const string EmailLabel = "E-mail";
+
     public static readonly ContactgegevenType Email =
         new(
-            nameof(Email)
+            EmailLabel
```

Comment:
Label opnemen in alle contactgegeven of dictionary bijhouden in projecties/acties

Reply:
voorstel Quinten: `Email = new (name: nameof(Email), label: "E-mail")` ?